subworkflows/run_histoqc.nf

```diff
@@ -8,7 +8,15 @@ workflow RUN {
 
     main:
 
-    params.convert ? CONVERT(images) | HISTOQC : HISTOQC(images)
+    def config_ch = params.custom_config ? file(params.custom_config) : params.config
+
+    if (params.convert) {
+        CONVERT( images ) 
+        HISTOQC( CONVERT.out, config_ch )
+    }
+    else {
+        HISTOQC ( images, config_ch )
+    }
```

Comment:
The error seems to suggest some behavioral difference between how config_ch is passed to HISTOQC on each side of this if/else statement.